### PR TITLE
[Compat]: Added advanced Netherite Compat

### PR DESCRIPTION
**Pull request by JorisDeBeer**

### CODE REVIEW

File: content/forgero-compat/src/main/resources/data/forgero/packs/advancednetherite/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "advancednetherite-compat",
+  "resource_type": "PACKAGE",
+  "priority": 4,
+  "namespace": "ae2",
```
Comment (SigmundGranaas) [inline content/forgero-compat/src/main/resources/data/forgero/packs/advancednetherite/package.json]: HMMMMMMM

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]
Looks good except mysterious namespace

File: content/forgero-compat/src/main/resources/data/forgero/packs/advancednetherite/package.json (hunk 1 above)
Comment (JorisDeBeer) [reply content/forgero-compat/src/main/resources/data/forgero/packs/advancednetherite/package.json]: I cry

**Review by SigmundGranaas — APPROVED** [review]
Looks great!

**Pull request merged**